Source/MediaInfo/Export/Export_Mpeg7.cpp

```diff
@@ -1053,15 +856,16 @@ Ztring Mpeg7_Transform_Visual(Ztring &ToReturn, MediaInfo_Internal &MI, size_t S
     }
     ToReturn+=Mpeg7_Visual_colorDomain(MI, StreamPos); //Algo puts empty string if not known
     ToReturn+=__T(">\n");
-    ToReturn+=__T("\t\t\t\t\t\t\t\t\t<mpeg7:Name xml:lang=\"en\">"); ToReturn+=Mpeg7_VisualCodingFormatCS_Name((VisualCodingFormatCS_termID/10000)*10000, MI, StreamPos); ToReturn+=__T("</mpeg7:Name>\n");
+    const Ztring CodeCS_Name = VideoCompressionCodeCS_Name((VisualCodingFormatCS_termID/10000)*10000, MI, StreamPos);
+    ToReturn+=__T("\t\t\t\t\t\t\t\t\t<mpeg7:Name xml:lang=\"en\">"); ToReturn+=CodeCS_Name; ToReturn+=__T("</mpeg7:Name>\n");
     if (VisualCodingFormatCS_termID%10000)
     {
         ToReturn+=__T("\t\t\t\t\t\t\t\t\t<mpeg7:Term termID=\"");
         ToReturn+=Ztring::ToZtring(VisualCodingFormatCS_termID/10000);
         ToReturn+=__T(".");
         ToReturn+=Ztring::ToZtring((VisualCodingFormatCS_termID%10000)/100);
         ToReturn+=__T("\">\n");
-        ToReturn+=__T("\t\t\t\t\t\t\t\t\t\t<mpeg7:Name xml:lang=\"en\">"); ToReturn+=Mpeg7_VisualCodingFormatCS_Name((VisualCodingFormatCS_termID/100)*100, MI, StreamPos); ToReturn+=__T("</mpeg7:Name>\n");
+        ToReturn+=__T("\t\t\t\t\t\t\t\t\t\t<mpeg7:Name xml:lang=\"en\">"); ToReturn+=CodeCS_Name; ToReturn+=__T("</mpeg7:Name>\n");

```

Comment:
Really same?
CodeCS_Name is `VideoCompressionCodeCS_Name((VisualCodingFormatCS_termID/10000)*10000` but here it is `Mpeg7_VisualCodingFormatCS_Name((VisualCodingFormatCS_termID/100)*100`

Reply:
Bug :-(